core/src/main/scala/dagr/core/cmdline/DagrCoreMain.scala

```diff
@@ -103,11 +109,15 @@ class DagrCoreArgs(
   @arg(doc = "Write an execution report to this file, otherwise write to the stdout")
   val report: Option[Path] = None,
   @arg(doc = "Provide an top-like interface for tasks with the give delay in seconds. This suppress info logging.")
-  var interactive: Boolean = false
+  var interactive: Boolean = false,
+  @arg(doc = "Use the experimental execution system.")
+  val experimentalExecution: Boolean = false,
```

Comment:
These changes include the later `execsystem2` changes as it was a PITA to break it up.